test/support/test_config.h

```diff
@@ -54,6 +54,8 @@
 #define _PSTL_STD_UNINITIALIZED_FILL_BROKEN (_MSC_VER == 1900)
 // GCC10 produces wrong answer calling exclusive_scan using vectorized polices
 #define TEST_GCC10_EXCLUSIVE_SCAN_BROKEN (_GLIBCXX_RELEASE == 10)
+// GCC7 std::get not returning const rvalue reference from const rvalue reference of tuple
+#define TEST_GCC7_RVALUE_TUPLE_GET_BROKEN (_GLIBCXX_RELEASE && _GLIBCXX_RELEASE < 8)

```

Comment:
To be consistent with the other macro in that config file, it make sense to rename to `_PSTL_TEST_GCC7_RVALUE_TUPLE_GET_BROKEN`

Reply:
Fixed